recipes/batchspawner/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set version = "1.1.0" %}
+
+package:
+  name: batchspawner
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/b/batchspawner/batchspawner-{{ version }}.tar.gz
+  sha256: 9bae72f7c1bd9bb11aa58ecc3bc9fae5475a10fdd92dc0c0d67fa7eb95c9dd3a
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps
```

` --no-deps` should be taken care off by `{{ PYTHON }}` or do you explicitly need it?
```suggestion
  script: {{ PYTHON }} -m pip install . -vv
```